frontend/src/components/VisaStatusOverview.js

```diff
@@ -0,0 +1,265 @@
+                </TooltipBox>
+            </Col>
+            <Col md={{ span: 5 }}>
+                <Space direction="horizontal">
+                    <TooltipBox title={t("overviewEmailIcon")}>
+                        <Button icon={<MailOutlined />} shape="circle" onClick={() => {}} />
+                    </TooltipBox>
+                    <TooltipBox title={t("overviewQQIcon")}>
+                        <Button icon={<QqOutlined />} shape="circle" onClick={() => {}} />
+                    </TooltipBox>
+                    <TooltipBox title={t("overviewAddtionalIcon")}>
+                        <Button icon={<EllipsisOutlined rotate={90} />} shape="circle" onClick={() => {}} />
+                    </TooltipBox>
+                </Space>
+            </Col>
+        </Row>
+    );
+};
+OverviewContentBar.propTypes = { ...overviewPropTypes.overview };
+
+const OveriviewHeaderBar = () => {
+    const [t] = useTranslation();
+    return (
+        <Row align="middle" className="ovreview-content-row" gutter={16}>
+            <Col xs={{ span: 0 }} md={{ span: 3 }}>
+                <strong>{t("Location")}</strong>
+            </Col>
+            <Col xs={{ span: 0 }} md={{ span: 4 }}>
+                <strong>{t("overviewEarliestDate")}</strong>
+            </Col>
+            <Col xs={{ span: 0 }} md={{ span: 4 }}>
+                <strong>{t("overviewLatestDate")}</strong>
+            </Col>
+            <Col xs={{ span: 0 }} md={{ span: 8 }}>
+                <strong>{t("overviewNewestFetch")}</strong>
+            </Col>
+            <Col xs={{ span: 0 }} md={{ span: 5 }}>
+                <strong>{t("overviewActions")}</strong>
+            </Col>
+        </Row>
+    );
+};
+
+const OverviewContentCard = ({ visaType, embassyCode, earliestDate, latestDate }) => {
+    const [t] = useTranslation();
+
+    return (
+        <Card
+            title={
+                <Row justify="space-between">
+                    <Col span={10}>{t(embassyCode)}</Col>
+                    <Space direction="horizontal">
+                        <Col span={3}>
+                            <TooltipBox title={t("overviewEmailIcon")}>
+                                <Button icon={<MailOutlined />} shape="circle" onClick={() => {}} />
+                            </TooltipBox>
+                        </Col>
+                        <Col span={3}>
+                            <TooltipBox title={t("overviewQQIcon")}>
+                                <Button icon={<QqOutlined />} shape="circle" onClick={() => {}} />
+                            </TooltipBox>
+                        </Col>
+                        <Col span={3}>
+                            <TooltipBox title={t("overviewAddtionalIcon")}>
+                                <Button icon={<EllipsisOutlined rotate={90} />} shape="circle" onClick={() => {}} />
+                            </TooltipBox>
+                        </Col>
+                    </Space>
+                </Row>
```

Comment:
Per [instruction of Grid from antd](https://ant.design/components/grid/#Outline)

> Your content elements should be placed directly in the col, and only col should be placed directly in row.

Will fix this in my commits today and I think I will refactor out the button groups for reusability